### ISSUE TITLE
Accessibility: calcite-dropdown container and enclosed button are clickable

### ISSUE THREAD

**Issue by MikeTschudi** [Esri/calcite-components#669]
which leads to the Firefox accessibility tool issue "Clickable elements must be focusable and should have interactive semantics"

![image](https://user-images.githubusercontent.com/2125181/84833934-2d43a180-afe5-11ea-8863-165fa019434f.png)

Is it possible for one of these to not be clickable?

**Comment by MikeTschudi** [comment]
At calcite-components version 1.0.0-beta.46, the latest on [unpkg.com](https://unpkg.com/@esri/calcite-components/dist/calcite/calcite.esm.js), appears to be fixed!

```html
<!doctype html>
<html lang="en" dir="ltr">
<head>
  <link rel="stylesheet" type="text/css" href="https://unpkg.com/@esri/calcite-components/dist/calcite/calcite.css" />
  <script type="module" src="https://unpkg.com/@esri/calcite-components/dist/calcite/calcite.esm.js"></script>
  <script nomodule="" src="https://unpkg.com/@esri/calcite-components/dist/calcite/calcite.js"></script>
</head>
<body>
  <!-- https://esri.github.io/calcite-components/?path=/docs/components-dropdown--simple -->
  <calcite-dropdown>
    <calcite-button slot="dropdown-trigger">Open Dropdown</calcite-button>
    <calcite-dropdown-group>
      <calcite-dropdown-item>Relevance</calcite-dropdown-item>
      <calcite-dropdown-item active>Date modified</calcite-dropdown-item>
      <calcite-dropdown-item>Title</calcite-dropdown-item>
    </calcite-dropdown-group>
  </calcite-dropdown>
</body>
</html>
```

**Comment by julio8a** [comment]
Still valid

**Comment by shoe913** [comment]
Still valid, @cosbyr, can we re-open?

**Comment by MikeTschudi** [comment]
Still present @1.0.0-beta.57

**Comment by MikeTschudi** [comment]
@1.0.0-beta.62, still found by Firefox' Accessibility developer tool

**Comment by MikeTschudi** [comment]
@1.0.0-beta.66, still found by Firefox' Accessibility developer tool

**Comment by MikeTschudi** [comment]
@1.0.0-beta.68, still found by Firefox' Accessibility developer tool

**Comment by MikeTschudi** [comment]
@1.0.0-beta.70, still found by Firefox' Accessibility developer tool

**Comment by MikeTschudi** [comment]
@1.0.0-beta.73, still found by Firefox' Accessibility developer tool

![image](https://user-images.githubusercontent.com/2125181/148840451-811b7415-0406-4bbf-8922-2f4dac8f58ce.png)

**Comment by jcfranco** [comment]
@MikeTschudi Apologies for the delay on this one, but can you check if this violation is flagged in Firefox by either:

1. NVDA or JAWS
2. [axe](https://www.deque.com/axe/), [WAVE](https://wave.webaim.org/) or [Lighthouse](https://addons.mozilla.org/en-US/firefox/addon/google-lighthouse/) plugins

?

I believe we are not supporting audits from the Firefox a11y tools at this time. @geospatialem Feel free to correct me.

**Comment by geospatialem** [comment]
@jcfranco Correct on support for accessibility, since browser tools have the potential to introduce errors. 

Performed some preliminary testing with NVDA and JAWS in Firefox and was able to access the dropdown and its contents successfully, including receiving feedback as to which dropdown item was checked. 

@MikeTschudi, can you recall if there were usability considerations with this one, in particular keyboard navigation experiences?

**Comment by MikeTschudi** [comment]
@geospatialem, keyboard navigation is fine. I created this issue solely because I've been trying to resolve all a11y issues in our apps. If this is a false positive, let's please close this issue. And good to know that I shouldn't test using the Firefox a11y tools.
